.github/workflows/linux.yml

```diff
@@ -58,19 +58,16 @@ jobs:
       MESON_ARGS: "-Dallow-noblas=true -Dcpu-baseline=none -Dcpu-dispatch=none"
     strategy:
       matrix:
-        version: ["3.11", "3.12", "3.13", "3.13t"]
+        version: ["3.11", "3.12", "3.13", "3.13t", "3.14-dev", "3.14t-dev"]
```

Comment:
Since free-threading is still rapidly evolving, perhaps remove `3.13t` already? It seems unlikely it would test anything not done by `3.14t-dev`.